### ISSUE TITLE
Convention joint axis for SMPL pose visualisation

### ISSUE THREAD

**Issue by antoinedelplace** [google/aistplusplus_api#11]
Hi there!
Hope you are well! :)

I am desperately trying to visualize the SMPL pose parameters from your dataset but it seems that your axis convention does not meet mine.

I have got inverted and inconsistent rotations as in the following picture :
![image](https://user-images.githubusercontent.com/34864698/109189406-474c9b00-7794-11eb-81ac-628abf2a4bae.png)

I have tried to swap (x, y z) axis, oppose signs in the x, y or z direction without success.

Do you have any idea of the transformation I need to make to visualize the animation properly ?
Could you please provide the axis convention used for your skeleton please ?

Thank you :D

**Comment by liruilong940607** [comment]
Hi! You can refer to the code [here](https://github.com/google/aistplusplus_api/blob/main/demos/run_vis.py#L77) about how to do proper SMPL forward kinematics. Instead of returning `.joints.detach().numpy()` from this `forward()` function, you can acquire SMPL mesh by returning `.vertices.detach().numpy()`. Please see [here](https://github.com/liruilong940607/smplx/blob/master/smplx/body_models.py#L393) about the output format from this forward kinematics function.

**Comment by antoinedelplace** [comment]
Hi!

Thank you for your response !

I have tried to get the vertices from the SMPL class but it seems like I get the same problem :
![vis_mesh](https://user-images.githubusercontent.com/34864698/109341236-674e8e00-786a-11eb-8b6e-43b454d3c556.gif)

I don't know where the problem could came from. Do you have any ideas ?

**Comment by liruilong940607** [comment]
It's hard to diagnose from what you show here. However, I just updated the repo with the support to visualize mesh, please see [here](https://github.com/google/aistplusplus_api#visualize-the-smpl-mesh).

It will interactively show the first frame of a sequence like this:
![Screenshot from 2021-02-27 13-28-19](https://user-images.githubusercontent.com/10151885/109400835-d629f680-78ff-11eb-92ff-e4f2068d5ae4.png)

**Comment by liruilong940607** [comment]
Update:

**It turns out the latest version of motion data is corrupted. The previous version is just fine. I'll investigate into it.**

**Comment by lj1995513** [comment]
how could I get the previous version of motion data?

**Comment by liruilong940607** [comment]
@lj1995513 We have fixed the corrupted data. The website links will be updated within two days and I'll post here.

**Comment by liruilong940607** [comment]
Now the links on the website have been fixed!
